jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/coder/OdeTimDataCreatorHelperTest.java

```diff
@@ -0,0 +1,46 @@
+package us.dot.its.jpo.ode.coder;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.jupiter.api.Test;
+import us.dot.its.jpo.ode.model.OdeTimData;
+import us.dot.its.jpo.ode.util.XmlUtils;
+import us.dot.its.jpo.ode.util.XmlUtils.XmlUtilsException;
+
+/**
+ * Tests for the OdeTimDataCreatorHelper class.
+ */
+public class OdeTimDataCreatorHelperTest {
```

Comment:
suggestion: [JUnit recommends leaving the modifier for most cases](https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-classes-and-methods). Removing the modifier will also get checkstyle to leave the warnings about missing Javadocs out of your test code

```suggestion
class OdeTimDataCreatorHelperTest {
```

Reply:
I think this is something that could probably be applied on a much wider scale than just the tests modified in this pull request. I did modify the OdeTimDataCreatorHelperTest with this request but I think it would be good to maybe do this across all of the tests.